### PR TITLE
CR-13: Disable certain routes for cloud

### PR DESCRIPTION
**Pull request by seth-riedel**

### CODE REVIEW

File: redisinsight/ui/src/components/main-router/config.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { isArray } from 'lodash'
+import { IRoute } from 'uiSrc/constants'
+
+export const LAZY_LOAD = window.riConfig?.app?.lazyLoad
```
Comment (seth-riedel) [inline redisinsight/ui/src/components/main-router/config.ts:4]: @ArtemHoruzhenko Is this the right way to use the new config?

File: redisinsight/ui/src/components/main-router/config.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+export const RI_ROUTES_EXCLUDED_BY_ENV = window.riConfig?.app?.routesExcludedByEnv
+
+export const getRouteIncludedByEnv = (routeDefinition: IRoute | IRoute[]) => {
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/components/main-router/config.ts:8]: let's always have an array
just replace in defaultRoutes
```getRouteIncludedByEnv({....})```
to
```getRouteIncludedByEnv([{....}])```

File: redisinsight/ui/src/components/main-router/config.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  if (RI_ROUTES_EXCLUDED_BY_ENV) {
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/components/main-router/config.ts:9]: did you check that the build is smaller with this env variable?

File: redisinsight/ui/src/components/main-router/config.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { isArray } from 'lodash'
+import { IRoute } from 'uiSrc/constants'
+
+export const LAZY_LOAD = window.riConfig?.app?.lazyLoad
+
+export const RI_ROUTES_EXCLUDED_BY_ENV = window.riConfig?.app?.routesExcludedByEnv
```
Comment (zalenskiSofteq) [inline redisinsight/ui/src/components/main-router/config.ts:6]: remove `RI_` please

File: redisinsight/ui/src/components/main-router/config.ts (hunk 1 above)
Comment (zalenskiSofteq) [reply redisinsight/ui/src/components/main-router/config.ts:4]: yes

File: redisinsight/ui/src/components/main-router/config.ts (hunk 3 above)
Comment (seth-riedel) [reply redisinsight/ui/src/components/main-router/config.ts:9]: The build size doesn't seem to be changing regardless of whether or not this variable is set

**Review by zalenskiSofteq — APPROVED** [review]

**Pull request merged**